src/helpers/_links.scss

```diff
@@ -10,7 +10,7 @@
 
 @mixin govuk-link-common {
   @include govuk-typography-common;
-  @include govuk-focusable-fill;
+  @include govuk-focusable-text-link;
```

Comment:
~~Links don't seem to be getting the focus text colour when in compatibility mode – happening on master as well so I don't think it was fixed by #1310 either?~~

![Screenshot 2019-05-03 at 09 41 10BST](https://user-images.githubusercontent.com/121939/57127398-35f8fa00-6d88-11e9-94b4-9f3addfe1b63.png)

Realised this was fixed by #1310, but on the v3 branch – can we rebase against `v3` to bring that in now it's been merged?